admin_site/system/views.py

```diff
@@ -1860,7 +1882,7 @@ def get_context_data(self, **kwargs):
             for pn in JSONSiteSummary.interesting_properties:
                 pv = getattr(p, pn)
                 # Don't convert these types to string representations...
-                if pv == None \
+                if pv is None \
```

Comment:
Now that you started to nicify the code it would be good to eliminate the backslash continuations here. Backslash continuations are evil™ when they can be avoided. 
Cf. https://pep8.org/: "The preferred way of wrapping long lines is by using Python’s implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation."